policy/release/authorization.rego

```diff
@@ -30,13 +30,14 @@ deny[result] {
 #   Enterprise Contract verifies if an authorized commit was used as the source of a build
 # custom:
 #   short_name: disallowed_commit_does_not_match
-#   failure_msg: Commit %s does not match authorized commit %s
+#   failure_msg: Commit %s does not match authorized commits
```

Comment:
Wondering if we would like to say what commits are authorized here for a bit better UX